### PR TITLE
Add example docstring for argwhere function

### PR DESCRIPTION
**Pull request by iamlordaubrey**

### CODE REVIEW

**Comment by iamlordaubrey** [comment]
Close 6351

**Comment by iamlordaubrey** [comment]
Hey @Ishticode, thanks for taking a look.

With respect to moving the examples for Container, would this refer to both the `Container` input and the `Container instance` examples?

**Review by Ishticode — CHANGES_REQUESTED** [review]
yes you are right about that.

**Comment by iamlordaubrey** [comment]
Hey @Ishticode, I already moved examples for Container to the argwhere instance method in ivy/Container/searching.py. 

I see some failing tests, but I think it has to do with some other files. Should I attempt fixing them or is it okay for those to fail within this MR?

**Review by Ishticode — CHANGES_REQUESTED** [review]
Hi the instance method example for the Array should be in the Array class mehtod argwhere.    The functional set of Container examples (not the instance one) can actually be placed in functional as they were (apologies for confusion) but you will need to remove the `ContainerWithSearching.argwhere(x)` part and replace it with just `ivy.argwhere(x)`

**Comment by iamlordaubrey** [comment]
Hi @Ishticode, thanks for clarifying, and I am sorry if I am still getting it wrong. 
My understanding is to move the instance method example (and container instance method example) to the class method argwhere (in ivy/Container/searching.py); while the functional set of examples should remain in the functional argwhere (as they originally were).

I've implemented this. Is this correct/okay?

File: ivy/container/searching.py [hunk 1]
```diff
@@ -424,6 +424,40 @@ def argwhere(
         -------
         ret
             Indices for where the boolean array is True.
+
+        Instance Method Examples
+        ------------------------
+
+        Using :class:`ivy.Array` instance method:
```
Comment (Ishticode) [inline ivy/container/searching.py:431]: Just need to move this example to ivy/array/searching.py under the argwhere method there.

**Review by Ishticode — CHANGES_REQUESTED** [review]
Nearly right. Just one change in the comment. Sorry for any confusion.

File: ivy/container/searching.py (hunk 1 above)
Comment (iamlordaubrey) [reply ivy/container/searching.py]: Moved this example to ivy/array/searching.py

**Review by Ishticode — CHANGES_REQUESTED** [review]
Hi Great stuff. One minor change and this is ready to merge. Can you keep the title as just `Examples` for all three cases.

**Review by Ishticode — CHANGES_REQUESTED** [review]

File: ivy/array/searching.py [hunk 2]
```diff
@@ -191,5 +191,20 @@ def argwhere(self: ivy.Array, *, out: Optional[ivy.Array] = None) -> ivy.Array:
         ret
             Indices for where the boolean array is True.
 
+        Examples
+        ------------------------
```
[the remark this answers is not in the document]
Comment (iamlordaubrey) [reply ivy/array/searching.py:195]: Oops... my bad. Sorry I missed that

**Pull request merged**